dev-docs/modules/userId.md

```diff
@@ -760,6 +760,11 @@ Bidders that want to support the User ID module in Prebid Server, need to update
                 "uids": [{
                     "id": "11111111"
                 }]
+            },{
+                "source": "netId",
```

Comment:
@phtechno - this is supposed to be a domain name.  How about enid.foundation ?

Reply:
never mind - found that the pbsBidAdapter code is setting netid.de. Will use that.